cdap-docs/admin-manual/source/installation/security.rst

```diff
@@ -112,34 +118,38 @@ Running Servers with SSL
-Property                                          Default Value
-================================================= ===========================================================================
-``router.bind.port``                              ``10000``
-``security.auth.server.bind.port``                ``10009``
-``dashboard.bind.port``                           ``9999``
-================================================= ===========================================================================
+================================================= ==================== ======================================================
+Property                                          Default Value        Description
+================================================= ==================== ======================================================
+``router.bind.port``                              ``10000``            Port number that the CDAP Router should bind to for 
+                                                                       HTTP Connections
+``security.auth.server.bind.port``                ``10009``            Port number that the CDAP Authentication Server should
+                                                                       bind to for HTTP
+``dashboard.bind.port``                           ``9999``             CDAP Console bind port
+================================================= ==================== ======================================================
 
 With SSL:
 
-================================================= ===========================================================================
-Property                                          Default Value
-================================================= ===========================================================================
-``router.ssl.bind.port``                          ``10443``
-``security.auth.server.ssl.bind.port``            ``10010``
-``dashboard.ssl.bind.port``                       ``9443``
-================================================= ===========================================================================
+================================================= ==================== ======================================================
+Property                                          Default Value        Description
+================================================= ==================== ======================================================
+``router.ssl.bind.port``                          ``10443``            Port number that the CDAP router should bind to for 
+                                                                       HTTPS Connections
+``security.auth.server.ssl.bind.port``            ``10010``            Port number that the CDAP Authentication Server should
+                                                                       bind to for HTTP
+``dashboard.ssl.bind.port``                       ``9443``             CDAP Console bind port
```

Comment:
should we make this consistent with the others?  "Port number that the CDAP Console should bind to for HTTP Connections" ?